.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
My question was clear: With `fast_finish`,  as soon as one job has failed, does a build finish immediatly (is stopped) or it continues to its normal end?